app/articles/serializers.py

```diff
@@ -81,3 +81,25 @@ def create(self, validated_data: Any) -> Any:
         article.tags.set(tags)
 
         return article
+
+
+class ArticleBookmarkSerializer(serializers.ModelSerializer):
+    """
+    Bookmarks serializer
+    """
+
+    user = UserSerializer(read_only=True)
+
+    class Meta:
+        model = ArticleBookmark
+        fields = (
+            "article",
+            "user",
+        )
+
+    def create(self, validated_data: Any) -> Any:
+        request = self.context["request"]
+        instance = ArticleBookmark.objects.get_or_create(
```

Comment:
As a BTW, another way you could have done this is using a hidden field with currentUserDefault